extrafiles/tooling/CppClangFormat.py

```diff
@@ -15,11 +15,22 @@
 import subprocess
 import os
 
+def isSupported(cmd):
+	""" Checks if the given clang-format command is available and if its version is recent enough. """
+	if shutil.which(cmd) is None:
+		return False
+	# Read major revision number from "clang-format version XX.XX.XX ... "
+	version = str(subprocess.check_output([cmd, "--version"],
+		universal_newlines=True)).strip().split()[2].split('.')[0]
```

Comment:
Make sure that this runs with `LANG=C` to force English output.

Reply:
Good point, I fixed that.

Reply:
Well, you added it below (which is good) but this check_output should also be changed ;)

Reply:
Ah I missed that, now it should be ok.